portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,20 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
```

Comment:
Can you order the imports based on https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing (ASCII sort order)?

Reply:
Can you recheck, although I think I resorted fine.